client/starwhale/core/eval/view.py

```diff
@@ -251,6 +252,7 @@ def run(
             name=name,
             desc=desc,
             step_spec=step_spec,
+            resource_pool=resource_pool,
```

Comment:
Does `EvaluationJob.run` really support `resource_pool` argument ?

Reply:
yes this pr supports it.
see `client/starwhale/core/eval/model.py`